src/DesignTime/InformationSchema.fs

```diff
@@ -148,7 +157,7 @@ type Column =
       AutoIncrement: bool
       DefaultConstraint: string
       Description: string
-      UDT: Type option Lazy // must be lazt due to late binding of columns with user defined types.
+      UDT: UDT option
```

Comment:
I'd suggest renaming `UDT` to `Enum` and defining `UDT` as:

```fsharp
type UDT =
    | Enum of Enum
    | Composite // to be implemented
```

I'm working on composite types at the moment (they already work in design-time actually) and this proposed change would go some way towards making it easier for my to integrate.

Reply:
UDT was removed too, as DataType type contains all information required to query for provided type from information schema cache

Reply:
I'm currently running integration tests / smoke testing our apps to ensure no regression was introduced, and will then review/merge other MRs. Without this, it's a bit hard to ensure that new functionality is not introducing regression, - a lot of tests within the TP repository where not working. Thanks for all the MRs, @kerams, can't wait to merge them!

Reply:
Yeah, having to comment out tests was indeed getting tedious. This is good stuff. I'll try to put together a script and a Dockerfile that will enable self-contained tests (like https://github.com/demetrixbio/FSharp.Data.Npgsql/issues/3#issuecomment-584158565), so that it's possible to set up CI for pull requests and whatnot.